.github/workflows/integration_tests_with_db.yaml

```diff
@@ -11,6 +11,18 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+
+      - name: Free Disk Space (Ubuntu)
+        uses: jlumbroso/free-disk-space@v1.3.0
+        with:
+          tool-cache: false
+          android: true
+          dotnet: true
+          haskell: true
+          large-packages: true
+          docker-images: true
+          swap-storage: false
+
```

Comment:
Did you list/request this action also [here](https://teamexasol.slack.com/archives/G8SB06C2K/p1697461011560019)?